waku/waku_test.go

```diff
@@ -1321,7 +1359,9 @@ func testConfirmationEvents(t *testing.T, envelope common.Envelope, envelopeErro
 		errorc <- err
 	}()
 	time.AfterFunc(5*time.Second, func() {
-		rw1.Close()
+		if err := rw1.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)